builtin/providers/aws/validators_test.go

```diff
@@ -553,3 +553,58 @@ func TestValidateDbEventSubscriptionName(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateJsonString(t *testing.T) {
+	type testCases struct {
+		Value    string
+		ErrCount int
```

Comment:
Since the cases in each slice below are all either valid or invalid, the `ErrCount` seems unnecessary, especially because if someone actually decided to add `ErrCount: 0` to `invalidCases` the error message wouldn't make much sense. But after all it's a test which is testing what it should, so 👍